client/blocks/conversation-follow-button/style.scss

```diff
@@ -72,13 +74,13 @@ button.conversation-follow-button {
 
 		&:hover,
 		&:focus {
-			color: var( --color-success );
+			color: var(--color-success);
 
 			.reader-following-conversation {
-				fill: var( --color-success );
+				fill: var(--color-success);
 				g {
 					fill: white;
-					stroke: var( --color-success );
```

Comment:
Are there any disable comments in the file which prevented these from being auto-fixed? (or maybe it's a shortcoming of the autofixer)

Reply:
No, the changes are from a recent PR where the dev doesn't seem to have the IDE properly set up and it looks like `--no-verify` flag was passed when committing the changes.

Reply:
Got it, thanks!